crates/re_space_view_spatial/src/ui_3d.rs

```diff
@@ -205,6 +205,15 @@ impl View3DState {
             self.orbit_eye = Some(target);
         }
     }
+
+    pub fn spin(&self) -> bool {
+        self.spin
+    }
+
+    pub fn set_spin(&mut self, spin: bool) {
+        self.spin = spin;
+        self.did_interact_with_eye = true;
```

Comment:
Just noting: as soon as we click spin, we'll lose the "zoom out to fit" behavior. I think this is a good compromise for now and a strict improvement as implemented, but it would be pretty neat if we could manage to get a behavior where it spins but also continues to zoom out as new data arrives.

Reply:
hmm yeah I suppose the "zoom to thing" method could also be made working with spin on. But let's leave this for another time!